be/src/storage/types.h

```diff
@@ -1425,6 +1430,22 @@ struct FieldTypeTraits<OLAP_FIELD_TYPE_OBJECT> : public FieldTypeTraits<OLAP_FIE
     }
 };
 
+template <>
+struct FieldTypeTraits<OLAP_FIELD_TYPE_JSON> : public FieldTypeTraits<OLAP_FIELD_TYPE_OBJECT> {
+    static Status convert_from(void* dest, const void* src, const TypeInfoPtr& src_type, MemPool* mem_pool) {
+        // TODO(mofei)
+        return Status::InternalError("Not supported");
+    }
+
+    static void copy_object(void* dest, const void* src, MemPool* mem_pool __attribute__((unused))) {
```

Comment:
Needn't implement it. we could remove this method

Reply:
fixed